docs/ux-guidelines-themes/theme-customization.md

```diff
@@ -0,0 +1,12 @@
+---
+post_title: WooCommerce Theme Guidelines - Customization
+menu_title: Customization
+---
+
+Themes have to rely on the customizer for any type of initial set up. Specific onboarding flows are not permitted.
```

Comment:
Hm this only applies to non-block themes. Would it be possible to add something like the following in helper text (or notice) at the top?

> This page of the guidelines applies to development of non-block themes only. For more specific guidance on development of block themes, refer to the [WordPress Developer's Guide to Block Themes](https://learn.wordpress.org/course/a-developers-guide-to-block-themes-part-1/).

Reply:
done!